backend/src/Book/Infrastructure/Application/BooksOperations.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Book\Infrastructure\Application;
+
+use App\Book\Domain\Entity\Book;
+use App\Book\Infrastructure\Repository\BookRepository;
+
+class BooksOperations
+{
+    public function __construct(private readonly BookRepository $bookRepository)
+    {
+    }
+
+    public function saveBook(float $price, string $author, string $title, string $description, Book $book = null): Book
```

Comment:
Hai ragionato sulla possibilità di creare due metodi distinti, `create` e `update`, per evitare il parametro opzionale e l'if per il check?
Non è un errore, è soltanto curiosità 🙂

Reply:
yes ci avevo ragionato, ho deciso di optarre per questa opzione perché creare due metodi distinti creava codice duplicato
diciamo che il mio pensiero è stato che il gioco non valeva la candela in questo caso perché il metodo mi sembrava già abbastanza esplicativo, seppur utilizzato per due scopi 😊